crates/mun_codegen/src/code_gen/symbols.rs

```diff
@@ -258,8 +258,9 @@ fn gen_function_info_array<'a, D: IrDatabase>(
     types: &AbiTypes,
     global_type_info_lookup_table: &HashMap<TypeInfo, PointerValue>,
     functions: impl Iterator<Item = (&'a hir::Function, &'a FunctionValue)>,
-) -> GlobalValue {
+) -> (GlobalValue, usize) {
```

Comment:
Maybe create a named struct here? Wasnt clear to me immediatly what the type ment.

Reply:
I will do that. Thanks!